suite-common/wallet-core/src/accounts/accountsThunks.ts

```diff
@@ -143,7 +143,7 @@ export const fetchAndUpdateAccountThunk = createThunk(
                 const areSatoshisUsed = getAreSatoshisUsed(bitcoinAmountUnit, account);
 
                 const formattedAmount = token
-                    ? `${formatAmount(token.amount, token.decimals)} ${token.symbol.toUpperCase()}`
+                    ? `${formatAmount(token.amount, token.decimals)} ${token.symbol?.toUpperCase()}`
```

Comment:
There's a possibility of having the `undefined` as a string inside this formatted amount. Please handle appropriately.

Reply:
fixed by: 562e424082